gen_binary.py

```diff
@@ -0,0 +1,66 @@
+from misc.viz_utils import visualize_instances_dict
+import numpy as np
+import json
+from skimage import io
+import cv2
+import zarr
+import sys, getopt, os
+import slideio
+
+'''
+Converts all json files in a folder to binary zarr.
+The corresponding svs files also need to be present to obtain height and width.
+All 3 files will have the same name.
+Sample call: python gen_binary.py -i /mnt/ibm_lg/snigdha/svs -o /mnt/ibm_lg/snigdha/svs
```

Comment:
do you mean for input and output paths to be same?

Reply:
Yes I find it easier to have all the files in the sample place.